framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java

```diff
@@ -518,6 +520,30 @@ private static boolean matchesAll(String actual, String expected) {
     return true;
   }
 
+  /**
+   * Check for regex matches.  The patterns can appear in any order in the
+   *  expected results file.
+   * 
+   * @param actual
+   *          string containing the actual results
+   * @param expected
+   *          string containing the expected results
+   * @return true or false to indicate if there is a match
+   */
+  private static boolean matchesAllNoOrder(String actual, String expected) {
+    String[] expectedLines = expected.split("\n");
+    actual = actual.trim();
+    int i = 0;
+    for (String string : expectedLines) {
+      string = string.trim();
+      Matcher matcher = Pattern.compile(string).matcher(actual);
+      if (!matcher.find()) {
```

Comment:
Also, if this works, I recommend we just modify the existing matchesAll() to accept a "checkOrder" boolean param.